srcpkgs/lightdm/template

```diff
@@ -13,7 +13,7 @@ license="GPL-3.0-or-later, LGPL-2.0-or-later"
 homepage="https://github.com/CanonicalLtd/lightdm"
 changelog="https://raw.githubusercontent.com/CanonicalLtd/lightdm/master/NEWS"
 distfiles="https://github.com/CanonicalLtd/lightdm/releases/download/${version}/lightdm-${version}.tar.xz"
-checksum=a118779650a3a5b20f05d53c4f9e377b484fbc1810a6b0914a60840d9a581f95
+checksum=12f5ab432748f0387c7cf8b94430495a558a035a4f8465e5181af6faff133e4b
 
```

Comment:
this empty line is unnecessary, and it's hiding some xlint issues

Reply:
Done, there were lots of xlint issues. They are now fixed.